docker/build_artifacts/sagemaker/tensorflow-serving.js

```diff
@@ -77,9 +77,10 @@ function tfs_json_request(r, json) {
             body = body.replace("\\'instances\\'", "'instances'")
         }
 
-        if ('application/jsonlines' == accept || 'application/jsons' == accept) {
+        var content_types = accept.trim().replace(" ", "").split(",")
```

Comment:
Could use recommendation on how / where to test this, if a test is necessary

Reply:
You could write a local integration test in here: https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/test/integration/local/test_container.py

More specifically, making a "predict" test and modify the content_types here: https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/test/integration/local/test_container.py#L74

Reply:
Fixed